src/js/jsx/search/SearchBar.jsx

```diff
@@ -229,10 +227,23 @@ define(function (require, exports, module) {
 
                         return toRemove ? toRemove.replace(" ", "") : idWord;
                     }, this).join("").toLowerCase(),
+                    partialMatch,
```

Comment:
Again, declarations without a definition should be moved to the end of var declaration blocks for stylistic consistency with the rest of the code.